### PR TITLE
OPENEUROPA-1945: Update allowed file extensions.

### PR DESCRIPTION
**Pull request by sergepavle**
### Description

The allowed file types can be found here: https://webgate.ec.europa.eu/CITnet/jira/browse/EWCMSREQ-315

From 1 to 35

Don't forget about the upgrade path.
### Change log

- Added:
- Changed:
- Deprecated:
- Removed:
- Fixed:
- Security:

### Commands

```sh
[Insert commands here]

```

### CODE REVIEW

File: tests/Kernel/FileExtensionTest.php [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+/**
+ * Tests a files upload of media bundles.
+ */
+class FileExtensionTest extends KernelTestBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public static $modules = [
```
Comment (upchuk) [inline tests/Kernel/FileExtensionTest.php]: protected

File: tests/Kernel/FileExtensionTest.php [hunk 2]
```diff
@@ -0,0 +1,59 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_media\Kernel;
+
+use Drupal\KernelTests\KernelTestBase;
+
+/**
+ * Tests a files upload of media bundles.
```
Comment (upchuk) [inline tests/Kernel/FileExtensionTest.php]: Tests that media types that allow file uploads use the correct extensions.

File: tests/Kernel/FileExtensionTest.php [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+  /**
+   * Tests a document upload restriction.
+   */
+  public function testDocumentUploadFileExtensions(): void {
+    $config = \Drupal::configFactory()->get('field.field.media.document.oe_media_file');
+    $this->assertEqual($config->get('settings.file_extensions'), 'txt text md readme info doc dot docx dotx docm dotm xls xlt xla xlsx xltx xlsm xltm xlam xlsb ppt pot pps ppa pptx potx ppsx ppam pptm potm ppsm pdf ods odt odf');
+  }
+
+  /**
+   * Tests a document upload restriction.
```
Comment (upchuk) [inline tests/Kernel/FileExtensionTest.php]: This is image

File: tests/Kernel/FileExtensionTest.php [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+    'image',
+    'file',
+    'system',
+    'oe_media',
+  ];
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function setUp($import_test_views = TRUE) {
```
Comment (upchuk) [inline tests/Kernel/FileExtensionTest.php]: Why import_test_views?

File: tests/Kernel/FileExtensionTest.php [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+    ]);
+
+  }
+
+  /**
+   * Tests a document upload restriction.
+   */
+  public function testDocumentUploadFileExtensions(): void {
+    $field = FieldConfig::load('media.document.oe_media_file');
+    $this->assertEqual($field->getSetting('file_extensions'), 'txt text md readme info doc dot docx dotx docm dotm xls xlt xla xlsx xltx xlsm xltm xlam xlsb ppt pot pps ppa pptx potx ppsx ppam pptm potm ppsm pdf ods odt odf');
```
Comment (brummbar) [inline tests/Kernel/FileExtensionTest.php]: `$this->assertEquals()` should be used. It's exactly the same, but `assertEqual` is marked as deprecated already. Same in the next test method.

**Pull request merged**